docs/sts/assume-role.md

```diff
@@ -35,7 +44,7 @@ XML error response for this API is similar to [AWS STS AssumeRole](https://docs.
 
 #### Sample Request
 ```
-http://minio:9000/?Action=AssumeRole&DurationSeconds=3600&Version=2011-06-15&AUTHPARAMS
+http://minio:9000/?Action=AssumeRole&DurationSeconds=3600&Version=2011-06-15&Policy={"Version":"2012-10-17","Statement":[{"Sid":"Stmt1","Effect":"Allow","Action":"s3:*","Resource":"*"}]}&AUTHPARAMS
```

Comment:
Resource needs to be in arn format - can you change "Resource":"" to "Resource":"arn:aws:s3:::" in this example?

Reply:
It doesn't need to be in ARN "*" is allowed.

Reply:
```
 aws --profile kp  --endpoint-url http://localhost:9000 sts assume-role --policy '{"Version":"2012-10-17","Statement":[{"Sid":"Stmt1","Effect":"Allow","Action":"s3:*","Resource":"*"}]}' --role-arn arn:aws:s3::::testbucket/blah.jpg --role-session-name kp   

An error occurred (InvalidParameterValue) when calling the AssumeRole operation: An invalid or out-of-range value was supplied for the input parameter.

```